iron-selection.html

```diff
@@ -99,8 +99,10 @@
       if (this.multi) {
         this.toggle(item);
       } else if (this.get() !== item) {
-        this.setItemSelected(this.get(), false);
-        this.setItemSelected(item, true);
+        this._mutateAtomic(function() {
```

Comment:
It overwrites `selectCallback` for the duration of the function so that it can call them all at once afterwards with only the final value. This is just to prevent the first call below to `setItemSelected` from triggering changes.